shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt

```diff
@@ -41,6 +41,10 @@ class MealApi {
     }
 
     companion object {
+        // iOS can use this endpoint
         private const val MEALS_ENDPOINT = "http://localhost:3000/Meals"
+
+        // Android needs to switch to this endpoint
+        // private const val MEALS_ENDPOINT = "http://10.0.2.2:3000/meals"
```

Comment:
What's this about?

Reply:
The android simulator won't resolve localhost:3000 so needed to change to the ip address shown.

Reply:
Super annoying